.snippets/code/tutorials/interoperability/remote-batched-evm-calls/terminal/encoded.md

```diff
@@ -0,0 +1,6 @@
+<div id="termynal" data-termynal>
```

Comment:
this file doesn't have the `<span class="file-path"></span` part, just shows up as `moonbeam@ubuntu-jammy:` instead of `moonbeam@ubuntu-jammy~$`